gravitee-apim-console-webui/src/management/api/portal/details/api-portal-details-danger-zone/api-portal-details-danger-zone.component.ts

```diff
@@ -69,6 +69,9 @@ export class ApiPortalDetailsDangerZoneComponent implements OnChanges, OnDestroy
   public isReadOnly = false;
 
   public get shouldUpgrade$() {
+    if (this.api.definitionVersion === 'V2') {
+      return false;
```

Comment:
The observable and value types must not be mixed. This makes the code difficult to maintain and creates bugs.

here, for example, even if isMissingFeature$ return `false`, the `*ngIf` will be evaluated as true.

I recommend typing the return of the function `shouldUpgrade$` like `shouldUpgrade$(): Observable<boolean>`
then to stay in the reactive mode and return `of(false)`

Reply:
same for the canStart$ 😱

Reply:
haha ok